app/src/main/java/fr/free/nrw/commons/WelcomeActivity.java

```diff
@@ -50,10 +48,17 @@ public void onCreate(Bundle savedInstanceState) {
             findViewById(R.id.finishTutorialButton).setVisibility(View.VISIBLE);
         }
 
-        ButterKnife.bind(this);
+        binding = ActivityWelcomeBinding.inflate(getLayoutInflater());
+        final View view = binding.getRoot();
+        setContentView(view);
+
+        pager = binding.welcomePager;
+        indicator = binding.welcomePagerIndicator;
```

Comment:
just a suggestion - this code is fine as it stands - but maybe these fields can go away and we use the appropriate fields out of the binding later on in the code?

Reply:
In app/src/test/kotlin/fr/free/nrw/commons/WelcomeActivityUnitTest.kt, we have:

    /**
     * Checks if the activity is not null and member variables are not null
     */
    @Test
    @Throws(Exception::class)
    fun checkActivityNotNull() {
        assertNotNull(activity)
        assertNotNull(activity.pager)
        assertNotNull(activity.indicator)
    }

This only passes when pager and indicator get instantiated. So, should the test be updated to remove the not-null assertions on pager and indicator, and we just remove the fields from the WelcomeActivity since the fields simply come out of the binding now?